app/src/main/java/org/oppia/android/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -304,4 +321,34 @@ class ProfileChooserFragmentPresenter @Inject constructor(
       activity.startActivity(pinPasswordIntent)
     }
   }
+
+  /** Handles navigation to the [AdministratorControlsActivity]. */
```

Comment:
/** Handles navigation to either the [AdministratorControlsActivity] or [AdminAuthActivity]. */ 

The modified KDoc provides more clarity.

Reply:
Updated.

Reply:
Okay.